python-flask-example-heroku/app.py

```diff
@@ -0,0 +1,10 @@
+import os
+from flask import Flask
+
+app = Flask(__name__)
+app.config.from_object(os.getenv("APP_SETTINGS", "config.DevelopmentConfig"))
+
+
+@app.route("/")
+def index():
+    return f"The configured secret key is {app.config.get('SECRET_KEY')}"
```

Comment:
The latest version from the tutorial looks like this:

```suggestion
import os
from flask import Flask

app = Flask(__name__)
env_config = os.getenv("APP_SETTINGS", "config.DevelopmentConfig")
app.config.from_object(env_config)


@app.route("/")
def index():
    secret_key = app.config.get("SECRET_KEY")
    return f"The configured secret key is {secret_key}."
```